Protocol.md

```diff
@@ -99,17 +70,73 @@ have no further frames sent and all frames will be ignored.
 
 ReactiveSocket as specified here only allows for TCP, WebSocket, and Aeron as transport protocols.
```

Comment:
Then it is somewhat confusing to then reference HTTP/2 in the later section about framing. Should that later mention be removed?